parsec/core/gui/authentication_change_widget.py

```diff
@@ -91,20 +105,20 @@ async def show_modal(cls, core, jobs_ctx, parent, on_finished=None):
                     hidden=True,
                 )
                 if not password:
-                    return
+                    return None
                 loaded_device = load_device_with_password(available_device.key_file_path, password)
             else:
                 loaded_device = await load_device_with_smartcard(available_device.key_file_path)
         except LocalDeviceError:
             show_error(parent, _("TEXT_LOGIN_ERROR_AUTHENTICATION_FAILED"))
-            return
+            return None
 
-        w = cls(core=core, jobs_ctx=jobs_ctx, loaded_device=loaded_device)
-        d = GreyedDialog(w, title=_("TEXT_CHANGE_AUTHENTICATION_TITLE"), parent=parent)
-        w.dialog = d
+        widget = cls(core=core, jobs_ctx=jobs_ctx, loaded_device=loaded_device)
+        d = GreyedDialog(widget, title=_("TEXT_CHANGE_AUTHENTICATION_TITLE"), parent=parent)
```

Comment:
Might as well change `d` to `dialog` for consistency

Reply:
done